### DISCUSSION TITLE
Getting an error.

### DISCUSSION THREAD

**Discussion by TroySeman** [quantumjot/btrack#319] [General]
I tried running with the default parameters and got an error.  I'm not sure if its my segmentation, my data, the parameters or a bug in the software.  This was a first run to see what happened with the defaults.

This is what I'm seeing in the console...

[INFO][2023/05/10 01:07:48 AM] Loaded btrack: C:\Users\Tinker\Anaconda3\envs\cellpose\lib\site-packages\btrack\libs\libtracker.DLL
01:07:48 INFO Loaded btrack: C:\Users\Tinker\Anaconda3\envs\cellpose\lib\site-packages\btrack\libs\libtracker.DLL
[INFO][2023/05/10 01:07:48 AM] Starting BayesianTracker session
01:07:48 INFO Starting BayesianTracker session
[INFO][2023/05/10 01:07:48 AM] Localizing objects from segmentation...
01:07:48 INFO Localizing objects from segmentation...
[INFO][2023/05/10 01:07:48 AM] Ending BayesianTracker session
01:07:48 INFO Ending BayesianTracker session
An error occurred while trying to format an error and show it in console.
You can try to uninstall IPython to disable rich traceback formatting
And/or report a bug to napari

**Comment by quantumjot** [comment]
Is this while trying to run the napari plugin? Also, can you provide a little more context about what you're trying to do? It'll help to debug

**Reply by TroySeman** [reply]
yes, the napari plug in.  I have used a cellpose model in the cellpose plugin to segment the image.  All the cells are touching in a zebra fish embryo with the nuclei labeled on the H1 histone.

**Reply by TroySeman** [reply]
![image](https://github.com/quantumjot/btrack/assets/19415564/bb92e247-49b1-4967-b285-3df2046411db)

**Reply by TroySeman** [reply]
![image](https://github.com/quantumjot/btrack/assets/19415564/ff0af0a8-5cfe-4159-b096-7a61005c1eb7)

**Reply by quantumjot** [reply]
Thanks for the update. However, I'm not sure what's causing these errors based on what you've provided.

If you run this example script, does it work?
https://gist.github.com/quantumjot/9d56dcd9a3487d7cb760f8421e18553c

**Comment by paddyroddy** [comment]
It looks like something to do with this https://forum.image.sc/t/napari-has-weird-output-when-starting-from-terminal/78047
